app/build.gradle

```diff
@@ -101,7 +101,7 @@ dependencies {
 
     implementation 'com.davemorrissey.labs:subsampling-scale-image-view-androidx:3.10.0'
     implementation 'com.github.MikeOrtiz:TouchImageView:3.1.1'
-    implementation 'com.github.ARK-Builders:ark-filepicker:ae2c37ea87'
+    implementation 'com.github.ARK-Builders:ark-filepicker:ed2ef717e4'
```

Comment:
This reference might change after ark-filepicker PR will be merged.

Reply:
> If I understand correctly, we need to merge [ARK-Builders/ark-filepicker#17](https://github.com/ARK-Builders/ark-filepicker/pull/17) first, and re-build APK for this branch then.

Yes